packages/cli/src/controllers/auth.controller.ts

```diff
@@ -73,19 +76,14 @@ export class AuthController {
 			if (preliminaryUser?.globalRole?.name === 'owner') {
 				user = preliminaryUser;
 			} else {
-				// TODO:SAML - uncomment this block when we have a way to redirect users to the SSO flow
-				// if (doRedirectUsersFromLoginToSsoFlow()) {
+				// TODO:SAML - implement option doRedirectUsersFromLoginToSsoFlow()
 				res.redirect(SamlUrls.restInitSSO);
```

Comment:
This is to make sure that if an owner tries to log in with username+password, even if SAML is enabled, they can still get in. For everyone else we redirect to SSO since that's where they should log in. We could also let them fail of course. It's a choice...